10up/ElasticPress
# Single sync option if index is deleted

### Description of the Change
This PR will address the issue where the Sync screen was showing 2 sync options along with the Last sync info. This will only show the Sync option which is visible during the initial syn.  


Closes #2962 

### How to test the Change

- In WP-CLI run wp elasticpress delete-index --index-name=$index_name --yes
- Visit the Sync page
- Observe that it shows single sync option without last sync info

### Changelog Entry

> Fixed - Sync page shows a successful previous sync even if the index has been deleted

### Credits

Props @MARQAS 


### Checklist:
